<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>SeniorTesterWindow</class>
  <widget class="QMainWindow" name="SeniorTester">
    <property name="geometry">
      <rect>
        <x>100</x>
        <y>100</y>
        <width>600</width>
        <height>600</height>
      </rect>
    </property>

  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>

    <property name="windowTitle">
      <string>Senior Tester</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QLabel" name="label">
            <property name="text">
              <string>Edit the Bugs</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignCenter</set>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QComboBox" name="comboBox_itemStatus">
            <property name="styleSheet">
              <string notr="true">QComboBox { font-size: 10px; }</string>
            </property>
            <item><property name="text"><string>yet-to-start</string></property></item>
            <item><property name="text"><string>in progress</string></property></item>
            <item><property name="text"><string>completed</string></property></item>
            <item><property name="text"><string>committed</string></property></item>
            <item><property name="text"><string>to test</string></property></item>
            <item><property name="text"><string>reopen</string></property></item> 
            <item><property name="text"><string>close</string></property></item>
          </widget>
        </item>

        <item>
          <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
              <size>
                <width>80</width>
                <height>30</height>
              </size>
            </property>
            <property name="text">
              <string>ok</string>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QTableView" name="tableView"/>
        </item>
        <item>
          <layout class="QHBoxLayout" name="buttonRowLayout">
            <item>
              <widget class="QPushButton" name="pushButton_create">
                <property name="minimumSize">
                  <size>
                    <width>80</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Create</string>
                </property>
              </widget>
            </item>

            <item>
              <spacer name="horizontalSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>

            <item>
              <widget class="QPushButton" name="pushButton_addFeature">
                <property name="minimumSize">
                  <size>
                    <width>80</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Update</string>
                </property>
              </widget>
            </item>

            <item>
              <spacer name="horizontalSpacer_2">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>

            <item>
              <widget class="QPushButton" name="pushButton_assign">
                <property name="minimumSize">
                  <size>
                    <width>80</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Assign</string>
                </property>
              </widget>
            </item>

            <item>
              <spacer name="horizontalSpacer_2">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>

            <item>
              <widget class="QPushButton" name="deleteButton">
                <property name="minimumSize">
                  <size>
                    <width>80</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Delete</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>

      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>